testv2/e2e/prow.yaml

```diff
@@ -0,0 +1,1428 @@
+# Code generated by e2e/generator, DO NOT EDIT.
```

Comment:
just a warning sign to the human if anyone would like to edit the file, some editors will warn them from doing so

Reply:
Sorry I've confused "file" and "line". That file eventually will land into .prow/autogenerated.yaml or something similar. At the moment it's intentionally written outside of the .prow to avoid any false launches but yet to see it materialized and to to see the generated diff delivered by the follow up PRs.